entrypoint.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
+service start cron
```

Comment:
@tuz666 did you read this https://github.com/lahmacunradio/arcsi/pull/177#issuecomment-1722394193? 

I still don't understand why tmpreaper should work without changing the docker compose file. I would suggest that we don't risk this (archon frontend) PR in any way (changing the entrypoint in an uncertain way is a risk) and that we experiment elsewhere. So I propose to remove this line before merging.

Reply:
@gammaw
I answered your questions under the previous PR. I wouldn't say that this change is a risk because I tested the `entrypoint.sh` change on the dev environment to make sure if it fix the tmpreaper issue or not, and as I checked this morning it worked as expected.

Reply:
Ok, thanks!